src/lib/components/billing/alerts/selectProjectCloud.svelte

```diff
@@ -0,0 +1,110 @@
+<script lang="ts">
+    import { type Models } from '@appwrite.io/console';
+    import { Alert, Button, Table } from '@appwrite.io/pink-svelte';
+    import { Modal } from '$lib/components';
+    import { onMount } from 'svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import { Dependencies } from '$lib/constants';
+    import { billingProjectsLimitDate } from '$lib/stores/billing';
+    import { page } from '$app/state';
+    import { toLocaleDateTime } from '$lib/helpers/date';
+
+    let { showSelectProject = $bindable(false) } = $props();
```

Comment:
```svelte
let {
    showSelectProject = $bindable(false),
}: {
    showSelectProject: boolean;
} = $props();
```

Reply:
making this change gave me an error
```
Property 'showSelectProject' does not exist on type '$$ComponentProps'.ts(2339)
```

Reply:
because you need another prop to pass this data. in the `org/+page.svelte` route, you are passing the projects array. but the component has a `$state` variable not an exported prop.

Reply:
something like this should work -
```ts
let {
    showSelectProject = $bindable(false),
    selectedProjects = [],
}: {
    showSelectProject: boolean;
    selectedProjects: string[];
} = $props();
```

Reply:
updated, thanks